sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/LatestFnTests.java

```diff
@@ -105,13 +109,15 @@ public void testAddInputSameTimestamp() {
 
   @Test
   public void testAddInputNullAccumulator() {
-    thrown.expect(NullPointerException.class);
+    thrown.expect(both(Matchers.<Throwable>instanceOf(NullPointerException.class))
```

Comment:
Wow, I totally thought two `.expect()` calls would overwrite each, but that's not the case. So much better.